### ISSUE TITLE
Noise Separation

### ISSUE THREAD

**Issue by munib94** [abcxyzi/RadSeg#1]
Hi,

Is there a way to separate the noise from the data?

**Comment by abcxyzi** [comment]
Yes, I hope to do this sometime in the future - I am assuming you're coming from the signal separation domain? But just in case, you may find the noise channel in the mask annotations useful. Hope it helps!
